tests_v2/test_workspace_create_layout.py

```
@@ -0,0 +1,285 @@
+                    "surfaces": [{
+                        "type": "terminal",
+                        "env": {"CMUX_LAYOUT_TEST_TOKEN": token},
+                    }],
+                },
+            },
+        })
+        c.select_workspace(ws)
+        surfaces = _surface_list(c, ws)
+        _must(len(surfaces) >= 1, f"expected at least 1 surface, got {len(surfaces)}")
+        surface_id = str(surfaces[0].get("id"))
+        c.send_surface(surface_id, f"printf 'CHECK=%s\\n' \"$CMUX_LAYOUT_TEST_TOKEN\"\\n")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)